java/ql/src/semmle/code/java/security/JWT.qll

```diff
@@ -0,0 +1,119 @@
+/** Provides classes for working with JWT libraries. */
+
+import java
+private import semmle.code.java.dataflow.ExternalFlow
```

Comment:
Unused? (same with the reverse import in `ExternalFlow`)

Reply:
Good catch, this is lingering code from when CSV models were removed. Fixed in a046d75, thanks!

Reply:
🤦  sorry